src/pages/Integrations/SplunkSetup/SplunkSetupPage.tsx

```diff
@@ -0,0 +1,366 @@
+import './SplunkSetup.scss';
+
+import {
+    ActionGroup,
+    Alert,
+    Button,
+    CodeBlock,
+    CodeBlockCode,
+    Flex,
+    FlexItem,
+    Form,
+    FormGroup,
+    Popover,
+    TextInput
+} from '@patternfly/react-core';
+import HelpIcon from '@patternfly/react-icons/dist/esm/icons/help-icon';
+import { Main, PageHeader, PageHeaderTitle, Section } from '@redhat-cloud-services/frontend-components';
+import axios from 'axios';
+import React, { useEffect, useState } from 'react';
+import { connect } from 'react-redux';
+import { Dispatch } from 'redux';
+
+import { Messages } from '../../../properties/Messages';
```

Comment:
@josejulio General observation: can properties be put on an import path? It could ease future refactoring should this be moved to a different path (or package).

Reply:
Of course. Thanks!